.github/workflows/plan-examples.py

```diff
@@ -0,0 +1,27 @@
+import json
+import glob
+import re
+
+
+def get_examples():
+    """
+    Get all Terraform example root directories using their respective `versions.tf`;
+    returning a string formatted json array of the example directories minus those that are excluded
```

Comment:
We need to document that `versions.tf` is required for examples.

Reply:
sure! What would be sufficient to cover this and where would it go?

Note: this logic is also the same logic used in https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/deec7d5caea47c06dea48fa616ad2c56e52c3cce/.github/workflows/pre-commit.yaml#L23